docs/framework-integration/vue.md

```diff
@@ -201,25 +184,33 @@ import { vueOutputTarget } from '@stencil/vue-output-target';
 export const config: Config = {
   namespace: 'stencil-library',
   outputTargets: [
+    // By default, the generated proxy components will
+    // leverage the output from the `dist` target, so we
+    // need to explicitly define that output alongside the
+    // Vue target
+    {
+      type: 'dist',
```

Comment:
By default, `npm init stencil` generates an output target configuration for `dist` with `loader` set to `loader: ../`. Can we add that to this section, so folks don't remove it assuming they need to match this section exactly?